internal/csi-addons/rbd/replication.go

```diff
@@ -280,15 +280,12 @@ func (rs *ReplicationServer) EnableVolumeReplication(ctx context.Context,
 	mgr := rbd.NewManager(rs.csiID, req.GetParameters(), req.GetSecrets())
 	defer mgr.Destroy(ctx)
 
-	rbdVol, err := mgr.GetVolumeByID(ctx, volumeID)
+	volumes, mirror, err := mgr.GetMirrorSource(ctx, volumeID, req.GetReplicationSource())
```

Comment:
let's call `volumeID` something else like reqID or replicationID.
It can then be either referring to a single volume or group (or later on snapshot).